### DISCUSSION TITLE
Group uploaded videos and assign group to a user

### DISCUSSION THREAD

**Discussion by gregk63** [mediacms-io/mediacms#1170] [Feature Requests]
**Describe the feature you'd like**
I am taking a look at mediaCMS as a possible solution. I have it installed and everything seems to be working but I am not seeing a way to group videos and then and assign that group to a user. This way a user only sees the videos that were assigned to them.

**Describe alternatives you've considered**
I tried creating a user, uploaded videos, created a category and was able to add videos to the category. I added a group but I am not seeing how to use the group.

**Additional context**
Am I missing something?

**Comment by gregk63** [comment]
Hey mgogoulos,

I could pay a little bit to get an answer on this! Interested in using this but I am having a hard time figuring out how to organizing videos in a way that fits my needs.

Thanks,

Greg

**Comment by mgogoulos** [comment]
This is a very custom workflow that you describe here. It's not supported out of the box, it would need development effort to implement it

**Comment by gregk63** [comment]
How big of development effort? Like I said I could pay a little bit as I am a company of one.
